frontend/src/views/UsersList.vue

```diff
@@ -0,0 +1,193 @@
+<template>
+  <div>
+    <div v-if="isLoadingLists">
+      <div class="elevation-1 pa-1">
+        <span :style="{ marginRight: '4px' }">Loading...</span>
+        <v-progress-circular indeterminate :size="15" :width="2"></v-progress-circular>
+      </div>
+    </div>
+
+    <v-data-table v-if="users.length"
```

Comment:
The DataTable component that wraps v-data-table is imported but not used. Did it not fit this use case? Also `:datatableProps` is specific to data-table not v-data-table. For v-data-table use `:loading`.